### DISCUSSION TITLE
stick to DTM-08 for now or wait until V2 is complete?

### DISCUSSION THREAD

**Discussion by tttMediumkey** [XYZenix/XYZenixThemes#81] [General]
Huge fan of DTM-08 since it's more relaxing to my eyes in comparison to skeuocord, I've heard that you're making a V2 version of the rewrite and as you said in the readme that it's still WIP. I haven't installed it to check it out but out of curiosity, should I wait until V2 is done or use it for the time being?

**Comment by XYZenix** [comment]
the rewrite is largely usable as it is right now...if you use dark mode, that is; i will finish light mode sooner or later but i don't expect it to look as good as dark mode

**Reply by tttMediumkey** [reply]
noted. ill give it a try and decide for myself later
